app-android/src/main/java/io/github/droidkaigi/confsched2022/KaigiApp.kt

```diff
@@ -81,21 +87,37 @@ fun KaigiApp(
                     navController = kaigiAppScaffoldState.navController,
                     startDestination = SessionsNavGraph.sessionRoute,
                 ) {
+                    val showNavigationIcon = !usePersistentNavigationDrawer
                     sessionsNavGraph(
+                        showNavigationIcon,
                         kaigiAppScaffoldState::onNavigationClick,
                         kaigiAppScaffoldState::onBackIconClick,
                         kaigiAppScaffoldState::onTimeTableClick,
                         kaigiAppScaffoldState::onNavigateFloorMapClick,
                     )
-                    contributorsNavGraph(kaigiAppScaffoldState::onNavigationClick)
-                    aboutNavGraph(kaigiAppScaffoldState::onNavigationClick)
+                    contributorsNavGraph(
+                        showNavigationIcon,
+                        kaigiAppScaffoldState::onNavigationClick,
+                    )
+                    aboutNavGraph(
+                        showNavigationIcon,
+                        kaigiAppScaffoldState::onNavigationClick,
+                    )
                     mapGraph()
                 }
             }
         }
     }
 }
 
+private val WindowSizeClass.usePersistentNavigationDrawer: Boolean
+    get() = when (widthSizeClass) {
```

Comment:
WindowSizeClass implements `Comparable<WindowWidthSizeClass>`.
So we can use like this `WindowWidthSizeClass.Medium < widthSizeClass` 👍 
I'll create the issue